worker-test/pom.xml

```diff
@@ -88,6 +88,35 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>integration-test</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>integration-test</goal>
+                        </goals>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <CAF_RABBITMQ_HOST>${docker.host.address}</CAF_RABBITMQ_HOST>
+                                <CAF_RABBITMQ_PORT>${rabbitmq.node.port}</CAF_RABBITMQ_PORT>
+                                <CAF_RABBITMQ_USERNAME>guest</CAF_RABBITMQ_USERNAME>
+                                <CAF_RABBITMQ_PASSWORD>guest</CAF_RABBITMQ_PASSWORD>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            
             <!-- Unpack the base worker configuration files -->
```

Comment:
Adding the surefire plugin with skip stops the ShutdownDeveloperTest.java running (its ignored anyway)
Adding the failsafe plugin enabled the existing It tests to run.

These run but fail connecting to RabbitMQ.

Adding the systemPropertyVariables enables connection to RabbitMQ, but tests not do not seem to be running. This is not the case in [US100117](https://github.com/WorkerFramework/worker-framework/pull/214) where the tests run but fail.